modules/gh-runner-gke/versions.tf

```diff
@@ -20,7 +20,7 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.3.0, < 7"
+      version = ">= 4.3.0, < 5.44"
```

Comment:
FYI: `"terraform-google-modules/kubernetes-engine/google//modules/beta-public-cluster/"` also uses the google-beta provider.